### ISSUE TITLE
Use same Emoji set like Android Client

### ISSUE THREAD

**Issue by webratte** [kontalk/desktopclient-java#20]
Use same Emoji set like Android Client.
The most features are available in Android Client should be available in Desktop Client I think.

**Comment by abika** [comment]
aahrr, I know this request would come at some point. Set to low priority delayed to whenever Im in the mood.

And when somebody has a library for having nice pictures in a JTextPlane I would be very glad ☺.

**Comment by webratte** [comment]
I think much more important then Emoijs are
[Share history among clients #250] (https://github.com/kontalk/androidclient/issues/250)

**Comment by webratte** [comment]
Can you use this for Desktopclient?:

https://www.soeren-hentzschel.at/firefox-os/das-ist-mozillas-open-source-emoji-font-fuer-firefox-os/

**Comment by abika** [comment]
no, thanks for looking, but unfortunately this has nothing to do with a Java/Swing library which we need here

**Comment by webratte** [comment]
OK.

I have found it by accident and thought maybe...

**Comment by k3b** [comment]
does this help ? http://stackoverflow.com/questions/9993424/how-to-add-images-in-jtextpane

**Comment by abika** [comment]
well, a solution would be good that doesn't require manually parsing the content text and replacing the unicode chars with images. But it looks a swing library for that doesn't exist and never will. 😞

**Comment by freddii** [comment]
i was able to see emojis in linux after installing ttf-ancient-fonts:

sudo apt install ttf-ancient-fonts

![emojis-kontalk](https://cloud.githubusercontent.com/assets/7213207/15935671/5dbdc0aa-2e68-11e6-9f47-623768a8d5aa.png)

**Comment by abika** [comment]
yes, that's the current status but unfortunately the spoiled users are used to fancy colored emojis like 🙈 or 🐅 and this shi*
